src/workloads/docs/Generators.yml

```diff
@@ -225,6 +229,10 @@ Actors:
                                              prid: { ^RandomDouble: { min: 0.0, max: 1000.0 } },
                                              data: { ^Join: { array: ["aaaaaaaaaa", {^FastRandomString: {length: {^RandomInt: {min: 0, max: 10}}}}]}}},
                                        number: 10}}
+            # You can choose elements deterministically. from is an array of values to pick from.
```

Comment:
[nit] s/from/'from' (generally, prefer to have named arguments in single quotes). Here and elsewhere.